lib/myfinance/resources/financial_account.rb

```diff
@@ -28,10 +26,33 @@ def create(entity_id, params = {})
       #   Documentation: https://app.myfinance.com.br/docs/api/receivable_accounts#delete_destroy
       #
       def destroy(id, entity_id)
-        http.delete(endpoints(id, entity_id, :destroy)) do |response|
+        http.delete(endpoint_for(id, entity_id, :destroy)) do |response|
           true
         end
       end
+
+      private
+
+      def request_and_build_response(method, endpoint, params={})
+        http.send(method, endpoint, body: { resource_key => params }) do |response|
+          respond_with_object(response, resource_key)
+        end
+      end
+
+      def endpoint_for(id, entity_id, key)
+        parameterize_endoint(id, entity_id, key)
+      end
+
+      def default_endpoints
+        {
+          create: "/entities/:entity_id/#{resource_key}s",
+          destroy: "/entities/:entity_id/#{resource_key}s/:id"
+        }
+      end
+
+      def parameterize_endoint(id, entity_id, key)
+        endpoints[key].gsub(':entity_id', entity_id.to_s).gsub(':id', id.to_s)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.